### ISSUE TITLE
Border Color is not perfect with radius

### ISSUE THREAD

**Issue by TapasFrontend** [bring-shrubbery/squircle-js#13]
![Screenshot 2024-04-26 at 10 08 55 AM](https://github.com/bring-shrubbery/squircle-js/assets/166890301/36c145ea-6304-4942-b3c2-81bfe6194f8c)

- Border color from the corner is not perfectly visible.

**Comment by iamdiogo** [comment]
An alternative is to render 2 squircles, one nested inside of the other, with the same `cornerRadius`.

**Comment by bring-shrubbery** [comment]
@iamdiogo Yeah, at the moment this is the solution, but I need to do that in-package and hide it under "borderWidth" prop. Will update here when that is implemented.

Answering the original question - the package uses `clipPath` to create the squircle effect. If you use regular border, that border will also get clipped.

**Comment by ariftesserasystems** [comment]
Hi there, any update on this beside having nested component?
